libs/utils/src/decorators/is-swif-v2-callback-url.decorator.ts

```diff
@@ -0,0 +1,27 @@
+import { registerDecorator, ValidationOptions, ValidationArguments } from 'class-validator';
+
+export function IsSwifV2CallbackUrl(validationOptions?: ValidationOptions) {
+  // eslint-disable-next-line func-names
+  return function (object: Object, propertyName: string) {
+    registerDecorator({
+      name: 'IsSwifV2CallbackUrl',
```

Comment:
Couldn't come up with a better generic name, so just went with the specific use case.